crates/loro-internal/src/handler.rs

```diff
@@ -1349,6 +1352,24 @@ impl TextHandler {
         }
     }
 
+    pub fn insert_utf8(&self, pos: usize, s: &str) -> LoroResult<()> {
+        match &self.inner {
+            MaybeDetached::Detached(t) => {
+                let mut t = t.try_lock().unwrap();
+                let index = t
```

Comment:
This should return error if the insert position is invalid

Reply:
I return a error in function `insert_with_txn_utf8`,is it wrong?

Reply:
On the 1361 line, in the Detached branch, `get_entity_index_for_text_insert` should return an Err if the given position is invalid.